src/cosima/src/MCSteppingAction.cc

```diff
@@ -118,6 +123,8 @@ MCSteppingAction::MCSteppingAction(MCParameterFile& RunParameters) :
   m_KnownProcess.push_back("LowEnConversion"); m_KnownProcessID.push_back(c_ProcessIDPair);
   m_KnownProcess.push_back("LowEnPolarizConversion"); m_KnownProcessID.push_back(c_ProcessIDPair);
   m_KnownProcess.push_back("muPairProd"); m_KnownProcessID.push_back(c_ProcessIDPair);
+  m_KnownProcess.push_back("ePairProd"); m_KnownProcessID.push_back(c_ProcessIDPair);
+
```

Comment:
new process now call `ePairProd` for e+e- pair production.
Same for `anti_lambdaInelastic` which probably replace `AntiLambdaInelastic`